docs/sources/auth/azuread.md

```diff
@@ -0,0 +1,103 @@
++++
+title = "Azure AD OAuth2 Authentication"
+description = "Grafana OAuthentication Guide "
+keywords = ["grafana", "configuration", "documentation", "oauth"]
+type = "docs"
+[menu.docs]
+name = "Azure AD"
+identifier = "azuread_oauth2"
+parent = "authentication"
+weight = 3
++++
+
+# Azure AD OAuth2 Authentication
+
```

Comment:
Please add more of an intro here to give the reader context. Tell the reader what to expect. Imagine you are looking at search results and ask yourself if this paragraph gives you enough information to know if the page contains the information you need. Right now, I do not think it does.

Reply:
I added some more context.